server/bundler/bin/bundler.js

```diff
@@ -106,4 +106,15 @@ webpack( webpackConfig, function( error, stats ) {
 	assets = utils.getAssets( stats.toJson() );
 
 	fs.writeFileSync( path.join( __dirname, '..', 'assets.json' ), JSON.stringify( assets, null, '\t' ) );
+
+	if ( ! process.env.WEBPACK_OUTPUT_JSON ) {
```

Comment:
oh, no, I see, this is currently there, but the whole section is marked as new because of the indent change?